bedrock/firefox/templates/firefox/developer-quantum-firstrun.html

```diff
@@ -126,73 +135,133 @@ <h3 class="body-title">{{ _('Build and Perfect your sites<br> with Firefox DevTo
           <li class="gallery-item feature">
             <img class="icon" src="{{ static('img/firefox/products/developer/quantum/feature-inspector.svg') }}" alt="">
             <h3 class="name">{{ _('Inspector') }}</h3>
-            <p class="desc">{{ _('Inspect and refine code to build pixel-perfect layouts.') }}</p>
+            <p class="desc">
+              {{ _('Inspect and refine code to build pixel-perfect layouts.') }}
+              {% if l10n_has_tag('quantum-firstrun-whatsnew') %}
+                <p class="features-learn-more"><a href="https://developer.mozilla.org/docs/Tools/Page_Inspector" rel="external">{{ _('Learn more about Page Inspector') }}</a></p>
```

Comment:
This nests a `p` inside a `p`. We could style the links with `display:block` themselves without the extra paragraph (same for all of them).

Reply:
Doh! Completely missed that.